### DISCUSSION TITLE
Thanks for this project!

### DISCUSSION THREAD

**Discussion by zkrzyzanowski** [PyO3/pyo3#2116] [Show and tell]
Our team recently converted a python codebase to rust and the performance increases have been incredible. This is our first experience with Rust and we've all been blown away by how great its been. Our package was mostly written by a data scientist with very little programming experience, and even just converting without refactoring any of the logic has seen our network requests go from 5-10 second range down to under .1s in most cases.

Just wanted to say thanks for all the work folks have put into this project and `rust-numpy`.

**Comment by davidhewitt** [comment]
Very pleased to hear you've had a great success with the project! It's results like this which motivate me to push PyO3 forward ☺️
